lib/opentok.js

```diff
@@ -901,6 +901,40 @@ OpenTok = function (apiKey, apiSecret, env) {
     });
   };
 
+  /**
+   * Send DTMF digits to all participants in an active OpenTok session or to a specific client
+   * connected to that session
+   *
+   * @param sessionId The session ID corresponding to the session to which the user will connect.
+   *
+   * @param connectionId The connection ID of the stream that the DTMF signal needs to be passed to.
+   *
+   * @param digits This is the string of DTMF digits to send. This can include 0-9, '*', '#', and
+   * 'p'. A p indicates a pause of 500ms (if you need to add a delay in sending the digits).
+   *
+   * @return Ok
```

Comment:
Can you do anything meaningful with the return value? If not, maybe the method should not return anything.

Also, please document the callback parameter. Something like: "The callback function invoked when the call to the method succeeds or fails. The callback function has two parameters: `error` (set if there is an error) and `json`..." What is included in the `json` upon success?

Reply:
Personally I don't like empty return calls just because a developer using an API usually expects a body to a request they make. However since the anvil api call doesn't return anything I guess it's okay to keep the pattern